app/uk/gov/hmrc/helptosavestub/controllers/CitizenDetailsController.scala

```diff
@@ -78,19 +78,21 @@ object CitizenDetailsController extends BaseController {
        dateOfBirth ← optionGen(dateGen())
     } yield Person(firstName, lastName, dateOfBirth)
 
+    val regexForValidNino = """^((GIR)(0AA)|([A-PR-UWYZ]([0-9]{1,2}|([A-HK-Y][0-9]|[A-HK-Y][0-9]([0-9]|[ABEHMNPRV-Y]))|""" +
```

Comment:
This is also defined in the companion object for `NSIUserInfo` - get the regex from there? Get rid of the `private` modifier on it if you have to